### DISCUSSION TITLE
Moving the DAO Index V0.9 Whitepaper Drafts to PubPub

### DISCUSSION THREAD

**Discussion by kylefox1** [Ledgerback/DGSF#5] [General]
Hi @linxule @vincenzospaghetti 

I will be moving my drafts for Whitepaper V0.9 from the repo to Ledgerback's PubPub.

It should be easier to edit the drafts over there.

Please let me know if you have any questions, comments, or other feedback.

**Comment by linxule** [comment]
Can we have it sync to the repo as well?

**Reply by kylefox1** [reply]
I am not sure how that would work, but I do not think it is necessary.
